debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSnapshotChangeEventSource.java

```diff
@@ -7,12 +7,18 @@
 
```

Comment:
Why is any change required in the Snapshot handling ? There are going to be no UPDATE records here

Reply:
This is required in order to maintain uniform record structure when published to Kafka since the transformer being used at the sink level won't be able to differentiate between snapshot or change records.

Reply:
if Replica Identity is a PG value (say DEFAULT) why should there be any change in Record Structure ? PG apps using PG values for Replica Identity should work with the record structure they expect from the PG Debezium connector

Reply:
Yes, they will work as it is. The method `YBSnapshotRecordEmitter#getNewColumnValues` only modifies the values if the replica identity is `CHANGE`, it maintains the original structure otherwise.